doc/onboarding.md

```diff
@@ -69,7 +75,8 @@ onboarding session.
       * When adding a semver label, add a comment explaining why you're adding it. Do it right away so you don't forget!
 
   * [**See "Who to CC in issues"**](./onboarding-extras.md#who-to-cc-in-issues)
-    * will also come more naturally over time
+    * For many of the teams listed there, you can just ask to be added if you are interested
```

Comment:
Micro-nit: remove `just`

Less micro: Remove `for many of the teams listed there`. People should freely ask. If they need to do something to join a WG or whatever, then it can be explained when they ask.